### ISSUE TITLE
Where to find Debug Logs Location?

### ISSUE THREAD

**Issue by necevil** [Interfacelab/ilab-media-tools#34]
Hey guys,
Trying to debug an issue where after entering my S3 storage credentials — and clicking import media — Media Cloud loops through each image and appears to upload, but no images ever show up in S3.

I assume it is a credential issue and have enabled debugging from the Plugin's list of options on the home landing screen but I am unsure of where to access these debug logs.  Hope you guys can help!

**Comment by jawngee** [comment]
You have to setup php error logging and it'll log to wherever you have that configured to in php.ini

**Comment by JulienMelissas** [comment]
Thanks for the quick reply @jawngee! That should help a lot with some of the issues I'm trying to debug here next week.

**Comment by JulienMelissas** [comment]
BTW, for the OP, code for the logger class is [here](https://github.com/Interfacelab/ilab-media-tools/blob/84a88c99132e3a23d420d7ab880718832ac7623c/classes/Utilities/Logger.php). Looks like we can use constants to define logging levels.

**Comment by necevil** [comment]
Hey right on.
It did end up being an authentication issue in my case.
I think there are some weird requirements as far as the types of S3 access that the plugin needs beyond just Put etc.

I was able to get it working by authorizing a new IAM user I created for ALL S3 plugins and didn't end up needing to enable logging — although it's great to know how to do that should that ever change.

Thanks @JulienMelissas @jawngee
